### ISSUE TITLE
Update Accumulo utilities to extend KeywordExecutable

### ISSUE THREAD

**Issue by cshannon** [apache/accumulo#2807]
Must utility applications extend [KeywordExecutable](https://github.com/apache/accumulo/blob/main/start/src/main/java/org/apache/accumulo/start/spi/KeywordExecutable.java). However, there are some that don't and these should be updated to do so. These applications are meant to be used with the `accumulo` command.

I did a search and here is a list of some examples that could be updated as part of this issue. They can be checked off as they get merged to mark as completed.

- [x] [DumpZookeeper](https://github.com/apache/accumulo/pull/2813)
- [ ] [LocalityCheck](https://github.com/apache/accumulo/pull/2808)
- [x] [ChangeSecret](https://github.com/apache/accumulo/pull/2814)
- [x] [DeleteZooInstance](https://github.com/apache/accumulo/pull/2815)
- [x] [RestoreZookeeper](https://github.com/apache/accumulo/pull/2816)
- [x] ~~[TableDiskUsage](https://github.com/apache/accumulo/pull/2809)~~
- [x] [TabletServerLocks](https://github.com/apache/accumulo/pull/2817)
- [x] [VerifyTabletAssignments](https://github.com/apache/accumulo/pull/2818)
- [x] [Cleanup Accumulo Admin utilities](https://github.com/apache/accumulo/pull/2881)

I will create PRs for these and submit them.

**Comment by cshannon** [comment]
I submitted PRs for a couple of these so far, I will continue with more this weekend. Not sure if there are any existing tests that need to be updated besides the one keyword I test but I can do a search as well.

**Comment by cshannon** [comment]
@dlmarion and @EdColeman - Ok this is ready for review, I went ahead and tackled 8 utilities for now, each with their own PR to review in isolation. I verified everything works using Uno and the accumulo command for each utility.

For the [UsageGroup](https://github.com/apache/accumulo/blob/main/start/src/main/java/org/apache/accumulo/start/spi/KeywordExecutable.java#L46) all the commands are just added under the default `OTHER` but this can be changed easily. Also some of these may be a candidate to be encapsulated under the [Admin](https://github.com/apache/accumulo/blob/main/server/base/src/main/java/org/apache/accumulo/server/util/Admin.java) command but for now I just made them their own commands.

**Comment by cshannon** [comment]
@milleruntime -

1. Testing - I ran all the commands against Uno and they _seemed_ to work but since I'm pretty new I wasn't sure how to verify the output on all of them. I looked around and didn't really see any existing unit tests for the utilities themselves to verify they work or functionality. Maybe another task should be to actually write some unit tests for these commands to verify they work (using mini accumulo, etc). I could create a new issue and PRs to add tests (if you agree they should be a separate issue)
2. I agree that I figured some of them would make more sense as part of the Admin command as I mentioned but wasn't sure which ones. I worked on 8 so far, I can updated my PRs and move ChangeSecret, Delete and Restore ZK to the Admin command. Any others you think?
3. I am also going to search around the code now and take a look now to see if there's anymore commands to fix but if you know of any others let me know.
